examples/mnist/README.md

```diff
@@ -0,0 +1,69 @@
+# MNIST
+
+
```

Comment:
Include a short description of the MNIST digit classifier task. You can see what we had written in the introduction:
https://docs.tensorleap.ai/guides-and-tutorials/mnist-guide